examples/com.mbta.rtr.lightrail-vehicle-information/LightRailVehicleInformation.json

```diff
@@ -0,0 +1,20 @@
+{
+  "data": {
+      "directionId": 1,
+      "lastAvi": "875",
+      "vehicleId": "G-10023",
+      "routeId": "Green-D",
+      "consist": ["3644", "3862"],
+      "orientation": "AB",
+      "nextStopId": "place-haecl",
+      "stopStatus": "IN_TRANSIT_TO",
+      "lat": "42.21469",
+      "lon": "71.03305",
+      "heading": "26.69"
+  },
+  "id": "1",
+  "source": "ctd.mbta.com/rtr-1234",
+  "specversion": "1.0",
+  "time": "2025-03-12T00:01:00.110000Z",
+  "type": "com.mbta.rtr.lightrail-vehicle-information"
```

Comment:
what do you all think of the naming here? Another idea was `lightrail-vehicle-heartbeat` to maintain parity with the subwayside vehicle heartbeat

Reply:
Hmm, what subwayside is describing seems quite distinct. what about `lightrail-movement`?

Reply:
i feel like `lightrail-movement` implies that these will only be sent when the vehicle moves. wouldn't you want to see  periodic updates even if the vehicle is not moving?

Reply:
Ah, yes. hmm. what about `update`? There's nothing specifically wrong with `information`, it's just slightly generic (and I don't want to miss an opportunity to make sure it's clear).

Reply:
Update works for me! I was searching for something better than information - generally I prefer events to have action verbs

Reply:
I renamed this event to `lightrail-vehicle-updated` and I renamed the other event to `lightrail-vehicle-deleted`. Using (object)-(past tense verb) makes these events match up nicely with the existing glides events in the schemas repo.